.gitignore

```diff
@@ -18,7 +18,5 @@ README.txt
 build/*
 dist/*
 .coverage
-samples/deviceFactory/localDeviceRegistry
```

Comment:
Can you restore these entries that got lost in a merge it seems

Reply:
Just committe that change, thanks